CWE/decisions/claim_greater.txt

```diff
@@ -5398,8 +5398,7 @@ NOT = { government = democracy government = hms_government }
 			badboy = 12
 			set_country_flag = claim_greater_europe
 			any_country = {
-				limit = { capital_scope = { continent = europe } }
-				all_core = { add_core = XEU }
+				any_owned = { limit = { NOT = { is_core = XEU } OR = { continent = europe } } add_core = XEU }
```

Comment:
I think it is better to use:

all_core = { limit = { NOT = { is_core = XEU } continent = europe } add_core = XEU } instead of any_owned